experimental/segmenter/Cargo.toml

```diff
@@ -35,7 +35,7 @@ serde_json = { version = "1.0", default-features = false, features = ["alloc"] }
 zerovec = { version = "0.9", path = "../../utils/zerovec", features = ["yoke"] }
 databake = { version = "0.1.0", path = "../../utils/databake", optional = true, features = ["derive"] }
 ndarray = { version = "0.15.5", default-features = false, optional = true, features = ["serde"] }
-num-traits = { version = "0.2", default-features = false, optional = true }
+num-traits = { version = "0.2", default-features = false, optional = true, features = ["libm"] }
```

Comment:
Question: Should we continue to use the `std` functions if the `std` feature is enabled? They might be more efficient or something.

Reply:
We are